tests/integration/cattletest/core/test_compose.py

```diff
@@ -1369,23 +1380,26 @@ def test_cert_removed(new_context, compose_bin, request):
                                       cert=CERT,
                                       certChain=CERT,
                                       key=KEY)
-    cert3 = client.create_certificate(name='cert2',
-                                      cert=CERT,
-                                      certChain=CERT,
-                                      key=KEY)
 
     cert = client.wait_success(cert)
     cert2 = client.wait_success(cert2)
-    cert3 = client.wait_success(cert3)
 
     assert cert.state == 'active'
     assert cert2.state == 'active'
-    assert cert3.state == 'active'
 
-    client.delete(cert2)
-    cert2 = client.wait_success(cert2)
+    cert2 = client.wait_success(cert2.remove())
```

Comment:
Can you do `client.wait_success(cert2.purge())` does that not work.  Otherwise this test takes awhile just waiting for the cert to be removed.